eStore/Core/Networking/ProductsRepository.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ProductsRepository.swift
+//  eStore
+//
+//  Created by Vladislav Kondrashkov on 11/17/19.
+//  Copyright © 2019 Vladislav Kondrashkov. All rights reserved.
+//
+
+import Foundation
```

Comment:
Hey Foundation, why are you here?

Reply:
![image](https://user-images.githubusercontent.com/11708104/69013147-eda0d280-098d-11ea-98a0-0198f2717ff0.png)